### DISCUSSION TITLE
Email a user/find owner of a username

### DISCUSSION THREAD

**Discussion by smythemm** [community/community#73213] [Other Feature Feedback, Questions, & Ideas]
My company wishes to set up a GitHub organization.

We can't use our company name (which is quite unique/distinctive) because there is already a user with this name.

It is possible that this is someone from my company and I would like to send them an email to ask if they would contact me. Is there any way to do this? I tried a "Forgot password" - but you need the email associated with the account.

I'm sure revealing this email would be a GDPR violation, but is there any other way to try reach out to the owner?
I considered logging a bug or something, but they have no public repos.

Any other assistance/ideas for me?

Thanks in advance.

**Comment by wuleninja** [comment]
Hi GitHub! I have the exact same issue. I cannot work with the name of my company, which is a Microsoft customer by the way. My company's name is taken with at least no public repos. It's possible it has been created by an employee that's not here anymore. How could we reclaim our rightful username ?

We're seriously considering going to GitLab at this point, we cannot set a bogus username just because it's been taken by a ghost.
